test/e2e_node/security_context_test.go

```diff
@@ -274,4 +276,84 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		})
 	})
 
+	Context("when creating containers with AllowPrivilegeEscalation", func() {
+		// parse the docker version
+		out, err := exec.Command("docker", "-v").CombinedOutput()
```

Comment:
There is a `ContainerRuntime` in `framework/TestContext` to make this conditional and effective only for docker test environments.